src/data/faq_de.json

```
@@ -465,11 +465,21 @@
-                            "Andererseits wird bei einer vorangegangenen Grundimmunisierung durch zwei Dosen die Auffrischimpfung als die 3. von 3 Impfungen deklariert. Durch den Eintrag „3 von 3“ erkennt die App die Auffrischimpfung korrekt als solche an und es entfällt die Anzeige der 14 Tage Wartezeit bis zum vollen Impfschutz. Sobald eine zweite Auffrischimpfung nach der Grundimmunisierung durch J&J durchgeführt wurde, wird die Anzeige der vierzehntägigen Wartezeit entfallen, da für dieses Zertifikat „3 von 3“ als Nummer der Impfung eingetragen wird.",
-                            "Als Nachweis können Sie für diese 14 Tage weiterhin Ihr Impfzertifikat für die Grundimmunisierung vorzeigen. Dieses behält seine Gültigkeit, sofern die technische oder fachliche Gültigkeit nicht abgelaufen ist."
+                            "Es gibt drei Fälle, in denen es zu einer Anzeige von einer vierzehntägigen Wartezeit nach einer Auffrischimpfung kommen kann:",
+                            "Fall 1: Die Grundimmunisierung erfolgt mit dem Impfstoff von Johnson & Johnson (J&J).",
+                            "Fall 2: Nach einer Genesung von COVID-19 erfolgt eine zweite Auffrischimpfung",
+                            "Fall 3: Nach einer Genesung von COVID-19 erfolgt eine erste Impfung mit J&J.",
+                            "Die Ursache liegt in der Limitierung der aktuellen EU Richtlinien zur Erstellung der verschiedenen Impfzertifikate/-reihenfolge, die durch die Apps entsprechend abgebildet wird. Siehe: <a href='https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_json_specification_en.pdf'>https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_json_specification_en.pdf</a> (auf Englisch)",
+                            "Begründung:",
+                            "Zu Fall 1 und Fall 2:",
+                            "Bei der Grundimmunisierung mit anderen Impfstoffen als J&J, also BioNTech, Moderna, oder AstraZeneca, wird im Zertifikat unter „Nummer der Impfung“ der Eintrag „2 von 2“ für die zweite Dosis vorgenommen. Da, unabhängig vom verwendeten Impfstoff, der volle Impfschutz aber erst nach der Ausbildung einer ausreichenden Menge von Antikörpern gegeben ist, wird nach dem letzten Impfvorgang der Grundimmunisierung immer eine Wartezeit von 14 Tagen gefordert.",
+                            "Dies hat zur Folge, dass Prüf-Apps, wie die CovPassCheck-App, nicht unterscheiden können, ob es sich beim zweiten Impfvorgang um eine Auffrischimpfung für die J&J-Grundimmunisierung , oder um die zweite Dosis der Grundimmunisierung handelt. Daher wird das Impfzertifikat für die Auffrischimpfung für 14 Tage als ungültig angezeigt.",

```

Remove space before comma (otherwise comma may go onto new line on its own).

```suggestion
                            "Dies hat zur Folge, dass Prüf-Apps, wie die CovPassCheck-App, nicht unterscheiden können, ob es sich beim zweiten Impfvorgang um eine Auffrischimpfung für die J&J-Grundimmunisierung, oder um die zweite Dosis der Grundimmunisierung handelt. Daher wird das Impfzertifikat für die Auffrischimpfung für 14 Tage als ungültig angezeigt.",
```